.github/stale.yml

```diff
@@ -0,0 +1,55 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels: []
+#  - pinned
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Label to use when marking as stale
+staleLabel: Review - Abandoned
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This PR has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs within 10 days.
```

Comment:
Instead of "recent activity", I think you should say specifically "activity in the last 30 days".

Reply:
Better mention the maintainers in the comment. I suggest adding `/cc @PowerShell/powershell-maintainers` at the end of the comment.